.gitignore

```diff
@@ -5,4 +5,21 @@ staticfiles/
 docs/_build
 *.sqlite3
 .tox
-.coverage
\ No newline at end of file
+.coverage
+/.vscode/
+node_modules/
+_layouts/
+/assets/generated/
+packages/webapp/build/
+_data/assets.json
+webapp-assets.json
+static/js
+/build
+
+npm-debug.log*
+yarn-debug.log*
+yarn-error.log*
+
+package-lock.json
+storybook-build
+yarn.lock
```

Comment:
Nope we don't want to ignore yarn.lock - very important to keep it under version control

```suggestion
```